web/twa-vis-framework/example-mapbox-vis/webspace/data/singapore/rail-lines.geojson

```diff
@@ -1 +1 @@
4757, 1.37879], [103.84768, 1.37845], [103.84778, 1.37816], [103.8479, 1.37766], [103.84796, 1.37725], [103.84846, 1.37326], [103.84856, 1.3726], [103.84862, 1.37225], [103.84876, 1.37178], [103.84905, 1.37117], [103.84965, 1.37011], [103.85011, 1.36929], [103.8503, 1.3689], [103.85053, 1.36826], [103.85061, 1.36805], [103.85292, 1.36395], [103.85308, 1.36352], [103.85319, 1.36311], [103.85323, 1.36286], [103.85326, 1.36251], [103.85325, 1.36208], [103.85318, 1.3616], [103.85309, 1.36125], [103.85296, 1.36093], [103.85287, 1.36068], [103.85267, 1.3603], [103.85251, 1.35996], [103.85243, 1.3598], [103.85224, 1.35943], [103.85205, 1.35899], [103.85191, 1.3587], [103.85171, 1.35831], [103.85157, 1.35807], [103.85126, 1.35762], [103.85084, 1.35708], [103.85061, 1.3568], [103.85044, 1.3566], [103.85021, 1.35631], [103.84986, 1.35575], [103.84972, 1.35546], [103.84941, 1.35494], [103.84922, 1.35461], [103.84896, 1.35406], [103.84886, 1.35378], [103.84872, 1.3533], [103.84854, 1.35244], [103.84837, 1.35172], [103.84832, 1.35144], [103.84824, 1.35107], [103.84818, 1.35082], [103.84813, 1.35058], [103.84799, 1.34994], [103.84783, 1.34918], [103.84771, 1.34868], [103.84766, 1.34844], [103.84689, 1.3447], [103.84683, 1.3412], [103.84683, 1.3406], [103.84682, 1.33988], [103.847, 1.33825], [103.84702, 1.33803], [103.84706, 1.33687], [103.84738, 1.3335], [103.84765, 1.33237], [103.84782, 1.33166], [103.84789, 1.33071], [103.84783, 1.32984], [103.84759, 1.32916], [103.84556, 1.32259], [103.84358, 1.32006], [103.84189, 1.31813], [103.84144, 1.31662], [103.84052, 1.31429], [103.83883, 1.313], [103.83854, 1.31277], [103.8341, 1.30965], [103.8334, 1.30811], [103.83219, 1.30529], [103.83206, 1.30488], [103.832, 1.30455], [103.832, 1.30439], [103.83223, 1.30327], [103.83229, 1.30294], [103.83268, 1.3021], [103.83306, 1.30135], [103.83349, 1.30092], [103.8342, 1.30061], [103.83467, 1.30048], [103.83524, 1.30032], [103.83599, 1.30042], [103.83605, 1.30044], [103.83669, 1.30063], [103.83715, 1.3006], [103.83851, 1.30034], [103.84026, 1.29994], [103.84285, 1.29933], [103.84502, 1.29878], [103.84604, 1.29851], [103.85065, 1.29552], [103.8512, 1.29493], [103.85179, 1.29401], [103.85199, 1.29372], [103.85235, 1.29337], [103.85262, 1.29309], [103.85318, 1.29248], [103.85355, 1.29177], [103.85365, 1.29126], [103.85357, 1.29082], [103.85257, 1.28815], [103.85193, 1.28668], [103.85171, 1.28606], [103.85161, 1.28544], [103.85155, 1.28465], [103.85154, 1.28441], [103.85146, 1.28297], [103.85149, 1.28278], [103.85152, 1.28266], [103.85176, 1.28233], [103.85208, 1.2818], [103.85262, 1.28086], [103.85318, 1.27995], [103.85375, 1.27899], [103.85404, 1.27846], [103.85418, 1.27813], [103.8543, 1.27768], [103.85475, 1.27517], [103.8549, 1.27468], [103.85506, 1.27431], [103.85523, 1.27399], [103.85558, 1.27348], [103.8558, 1.27323], [103.85818, 1.27102], [103.85897, 1.27071], [103.85963, 1.27062], [103.86038, 1.27058], [103.86104, 1.27071], [103.86141, 1.27087], [103.86272, 1.27175]]}}]}
```

Comment:
I can't see a reason for the change in this file, please list reason or revert.

Reply:
This was an accidental result of a find and replace of 3838 -> 38383... Reverting!